templates/web/smidsy/tokens/confirm_problem.html

```diff
@@ -0,0 +1,60 @@
+[% INCLUDE 'header.html', bodyclass = 'fullwidthpage', title = loc('Confirmation') %]
+[% SET s = c.cobrand.get_severity(report.extra.severity) %]
+[% SET police = c.cobrand.get_police_info(report) %]
```

Comment:
Would it not make sense to only make this network call if the conditions below are met?